src/main/resources/config/application-dev.yml

```diff
@@ -38,7 +38,7 @@ spring:
     type: com.zaxxer.hikari.HikariDataSource
     url: jdbc:mysql://localhost:3306/academyProject?useUnicode=true&characterEncoding=utf8&useSSL=false&useLegacyDatetimeCode=false&serverTimezone=UTC
     username: root
-    password:
+    password: georgiana
```

Comment:
Pe viitor, sa incerci sa nu-ti mai dai commit la parolele pe care le folosesti pe localhost

Reply:
Fisierul asta nu ar trebui comis, pt ca va genera conflicte. Poti sa stergi randul asta modificat, adica sa rescri in fisier password:  sa mai faci un commit si apoi sa il modifici la tine local, fara sa il mai comiti. Poti sa cauti pe net sau daca nu Iti arat data viitoare cum sa iti faci un changelist separat pt chestii care nu trebuie comise :)